.github/workflows/stale_issues.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Stale issue job
     steps:
-    - uses: aws-actions/stale-issue-cleanup@v3
+    - uses: aws-actions/stale-issue-cleanup@main
```

Comment:
https://github.com/aws-actions/stale-issue-cleanup#readme recommends pinning an explicit tag or commit
> NOTE: For stability, you should use the action with either an explicit tag, or commit SHA

Should we follow our own advice here? If not, why diverge?

Reply:
If that is the case we should update the guidance in the README.